packages/editor/src/components/browser/roleSelector/RoleSelectorBrowser.tsx

```diff
@@ -0,0 +1,132 @@
+import { useEffect, useMemo, useState } from 'react';
+import { SelectRow, Table, TableCell, ExpandableCell } from '../../widgets';
+import type { UseBrowserImplReturnValue } from '../useBrowser';
+import { IvyIcons } from '@axonivy/editor-icons';
+import type { BrowserValue } from '../Browser';
+import {
+  useReactTable,
+  type ColumnDef,
+  type RowSelectionState,
+  getCoreRowModel,
+  getFilteredRowModel,
+  flexRender,
+  type ExpandedState,
+  getExpandedRowModel
+} from '@tanstack/react-table';
+import { useRoles } from '../../parts/common/responsible/useRoles';
+import type { RoleMeta } from '@axonivy/inscription-protocol';
+export const ROLE_SELECTOR_BROWSER = 'role' as const;
+
+export type RoleSelectorOptions = {
+  showtaskRoles?: boolean;
```

Comment:
seems that this something I didn't well. Please rename this property to `showTaskRoles`